benchmark/fluid/run_dist_benchmark_example.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+
+PADDLE_TRAINING_ROLE=PSERVER PADDLE_PSERVER_PORT=7164 PADDLE_PSERVER_IPS=127.0.0.1 PADDLE_TRAINERS=2 PADDLE_CURRENT_IP=127.0.0.1 PADDLE_TRAINER_ID=0 python fluid_benchmark.py --model resnet --device GPU --update_method pserver --iterations=10000 &
```

Comment:
Seems the style check on travis ci didn't pass. And, I think maybe it's a good idea to put this in the `README.md`, for a distributed training example?

Reply:
sure. perhaps tell user to start ps first and maybe sleep for a while?